### PR TITLE
Generate the demographic distribution graphs using the same logic as the csv

### PR DESCRIPTION
**Pull request by as2388**
Review #72 first.

Sample output: https://drive.google.com/drive/folders/1VqRDbQUgYaX25cm1Iet8k1YJy6PFeNZP?usp=sharing

### CODE REVIEW

**Comment by IsaackMwenda** [comment]
@as2388 would you be having examples of the graphs generated by this script?

File: generate_analysis_graphs.py [hunk 1]
```diff
@@ -206,12 +254,30 @@
         title="Participants per Episode"
     ).save(f"{output_dir}/participants_per_episode.png", scale_factor=IMG_SCALE_FACTOR)
 
+    log.info("Graphing the demographic distributions...")
+    for demographic, counts in demographic_distributions.items():
+        log.info(f"Graphing the distribution of codes for {demographic}...")
+        altair.Chart(
+            altair.Data(values=[{"code_string_value": code_string_value, "number_of_individuals": number_of_individuals}
+                                for code_string_value, number_of_individuals in counts.items()])
+        ).mark_bar().encode(
```
Comment (IsaackMwenda) [inline generate_analysis_graphs.py:263]: We should add data labels here to make the charts easy to interpret.

**Review by IsaackMwenda — CHANGES_REQUESTED** [review]
Requesting changes due to the chart labels.

**Comment by as2388** [comment]
> @as2388 would you be having examples of the graphs generated by this script?

Added to the description. Apologies for missing this before.

File: generate_analysis_graphs.py (hunk 1 above)
Comment (as2388) [reply generate_analysis_graphs.py:263]: What do you mean here? The graphs already have axis titles and labels.

Comment (IsaackMwenda) [reply generate_analysis_graphs.py:263]: I meant adding data values for each bar e.g https://altair-viz.github.io/gallery/bar_chart_with_labels.html

**Comment by IsaackMwenda** [comment]
> > @as2388 would you be having examples of the graphs generated by this script?
> 
> Added to the description. Apologies for missing this before.

Thanks : )

File: generate_analysis_graphs.py (hunk 1 above)
Comment (lukechurch) [reply generate_analysis_graphs.py:263]: I don't these are necessary, we probably don't want people caring about whether something is 23 or 26 given the kind of data that AVF uses

**Review by IsaackMwenda — APPROVED** [review]

**Review by lukechurch — APPROVED** [review]

**Pull request merged**